libs/ui-toolkit/src/components/toggle/toggle.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import type { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Toggle } from './toggle';
+import { useState } from 'react';
+
+export default {
+  component: Toggle,
+  title: 'Toggle',
+} as ComponentMeta<typeof Toggle>;
+
+export const Controlled: ComponentStory<typeof Toggle> = () => {
+  const [checked, setChecked] = useState('test-1');
+
+  return (
+    <form>
```

Comment:
suggestion (non blocking): Might be nice to render the current selected state here to demonstrate controlled mode.

Reply:
Good idea, thanks